zbarcam/zbarcam.py

```diff
@@ -79,6 +79,11 @@ def _detect_qrcode_frame(self, instance, camera, texture):
         image_data = texture.pixels
         size = texture.size
         fmt = texture.colorfmt.upper()
+        # PIL doesn't support BGRA but IOS uses BGRA for the camera
+        # if BGRA is detected it will switch to RGBA, color will be off
+        # but we don't care as it's just looking for barcodes
+        if fmt == 'BGRA':
```

Comment:
Should we `if platform == 'ios' and fmt == 'BGRA':` so we're sure we don't break things for other platforms?
What if for example `RGBA` is not supported in one of the other platforms?